opensearch/src/main/java/org/opensearch/sql/opensearch/request/ContinuePageRequestBuilder.java

```diff
@@ -5,94 +5,39 @@
 
 package org.opensearch.sql.opensearch.request;
 
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
 import lombok.Getter;
-import org.apache.commons.lang3.tuple.Pair;
 import org.opensearch.common.unit.TimeValue;
-import org.opensearch.index.query.QueryBuilder;
-import org.opensearch.search.aggregations.AggregationBuilder;
-import org.opensearch.search.sort.SortBuilder;
-import org.opensearch.sql.ast.expression.Literal;
 import org.opensearch.sql.common.setting.Settings;
-import org.opensearch.sql.expression.ReferenceExpression;
-import org.opensearch.sql.opensearch.data.type.OpenSearchDataType;
 import org.opensearch.sql.opensearch.data.value.OpenSearchExprValueFactory;
-import org.opensearch.sql.opensearch.response.agg.OpenSearchAggregationResponseParser;
 
 /**
  * Builds a {@link ContinuePageRequest} to handle subsequent pagination/scroll/cursor requests.
- * Initial search requests is handled by {@link InitialPageRequestBuilder}.
+ * Initial search requests are handled by {@link OpenSearchRequestBuilder}
  */
-public class ContinuePageRequestBuilder extends PagedRequestBuilder {
+public class ContinuePageRequestBuilder implements ExecutableRequestBuilder {
```

Comment:
I thought the request was to remove this completely?  We still want to keep this around in a reduced form?

Reply:
Request was to remove `OpenSearchPagedIndexScan` and use only `OpenSearchIndexScan`. 

Internally, `OpenSearchIndexScan` already used two different request types -- `OpenSearchScrollRequest` and `OpenSearchQueryRequest`. 

`ContinuePagedRequest` is the third one. It has a different use case compare to OpenSearchScrollRequest -- a continuation of a scroll request.